frontend/components/App/App.tsx

```diff
@@ -22,9 +22,14 @@ import Fleet404 from "pages/errors/Fleet404"; // @ts-ignore
 import Fleet500 from "pages/errors/Fleet500";
 import Spinner from "components/Spinner";
 
+interface IRouter extends InjectedRouter {
+  location: {
+    pathname: string;
+  };
+}
 interface IAppProps {
   children: JSX.Element;
-  router: InjectedRouter;
+  router: IRouter;
```

Comment:
You can replace this with `location`. It's a prop already passed from react-router. The type has to be `any`, however, since react-router v3 doesn't have a type for it.

Reply:
Ah, this would be a good reason to upgrade `react-router`. No type support!

Reply:
Yeah...It took me a while to track down the types they do have. v3 is SO OLD lol.

Reply:
Ah, I forgot about `location`! It's been a long day lol